libraries/RC_Channel/RC_Channels.cpp

```diff
@@ -89,6 +90,20 @@ bool RC_Channels::read_input(void)
         success |= channel(i)->update();
     }
 
+    // check if clear overrides by RC is allowed by RC_OPTIONS and any change in RC input during RC overrides
```

Comment:
create a subroutine for this

Reply:
I've added the `should_clear_overrides` subroutine as suggested.